src/sqlfluff/core/parser/segments/raw.py

```diff
@@ -75,14 +79,7 @@ def make(cls, template, case_sensitive=False, name=None, **kwargs):
         name = name or _template
         # Now lets make the classname (it indicates the mother class for clarity)
         classname = "{0}_{1}".format(name, cls.__name__)
```

Comment:
I'm wondering whether we *need* to make this classname. I'm not sure what the implications of ditching it would be, but what if all of our `KeywordSegment` classes just still had the name `KeywordSegment`? That would keep things simpler.

It could also cause a big mess, so it's a pain then feel free to forget it.